meal-planner/src/components/Button.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components'
+import slate from '../assets/slate.jpg'
+
+export default function Button({ day, meal, onClick, id }) {
+    return (
+        <ButtonStyled id={id} onClick={onClick}>
+            <span className={'day'}>{day}</span>{' '}
```

Comment:
What does {' '} do?

Reply:
I have no idea :-) somehow autmaticly generatetd. How ever, I deleted it.